app/forms/work_deposit_pathway.rb

```diff
@@ -228,9 +228,21 @@ class ContributorsFormBase < WorkVersionFormBase
                :contributor=,
                to: :work_version, prefix: false
 
+      validate :validate_creator_names
+
       def form_partial
         'non_instrument_work_version'
       end
+
+      private
+
+        def validate_creator_names
+          creators.each do |creator|
+            if creator.given_name.blank? || creator.surname.blank?
+              errors.add(:creators, 'Each creator must have a given name and a surname.')
```

Comment:
I forgot to bring this back up while we were pairing;  We should store the text for the error message in the locales.  Probably somewhere in here: https://github.com/psu-libraries/scholarsphere/blob/main/config/locales/en.yml#L510-L511